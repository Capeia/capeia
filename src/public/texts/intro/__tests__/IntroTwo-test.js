// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<IntroTwo>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/intro/IntroTwo')
    const Component = require('public/texts/intro/IntroTwo').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
