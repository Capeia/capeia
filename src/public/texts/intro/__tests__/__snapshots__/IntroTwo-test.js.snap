// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroTwo> renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="contentContainer  col-lg-6 col-lg-offset-3 col-md-8"
    >
      <div
        className={undefined}
      >
        <div
          className=""
        >
          <h1>
            Capeia is not...
          </h1>
          <h2>
            ...a conventional science news portal
          </h2>
          <p>
            Our authors are all active scientists involved in competitive research and are specialists in their own field. We do not adopt text from news agencies or other media. Each article is a genuine original contribution that draws on the author‚Äôs critical faculty and experience accumulated over a career in academic research.
          </p>
          <p>
            Articles published on Capeia reflect the dynamic nature of science, placing a focus not only on its great moments but also on its pitfalls, ambiguities and uncertainties. Today‚Äôs scientific industry churns out, at least initially, vast quantities of data that are in desperate need of interpretation and discussion before any new knowledge can be achieved.
          </p>
          <p>
            Capeia is committed to improving our understanding of scientific matters by picking up interesting pieces of information from different sources, including research publications, media reports, book releases, and conferences, and commenting on them in a critical manner that can be easily understood by non-experts.
          </p>
          <p
            className="iconSeparator"
          >
            <img
              src="sections/zoology.svg"
            />
          </p>
          <h2>
            ...a conventional crowdfunding platform
          </h2>
          <p>
            Every scientist registered on Capeia is provided with a forum in which to present his or her research, together with the opportunity to receive funding from the community. On Capeia, however, there is no need for scientists to run a frantic online campaign that may or may not yield a pre-defined amount of funding. We believe that our scientists should devote their energy to what they do best: advancing their field of research and sharing their thoughts and reflections with the rest of us. On Capeia, financial support for scientists is provided on a ‚Äúcontinuous funding for continuous research‚Äù basis. The more funding a scientist receives the more he or she can advance in his or her field by recruiting students, buying cool equipment, or having expensive analyses carried out. This funding scheme means that money once donated won‚Äôt come back to you, so better think carefully before pressing the support button üòâ. All donations, minus a fee for expenses involved in maintaining and running the platform, are paid into an account held by the scientist's institute. From there, the money gets routed directly to the scientist's research account.
          </p>
          <p
            className="iconSeparator"
          >
            <img
              src="sections/planetary_science.svg"
            />
          </p>
          <h2
            id="not-a-conventional-social-media-platform"
          >
            ...a conventional social media platform
          </h2>
          <p>
            Science is a lot of fun and it thrives on interactions between curious minds. We cordially invite you to participate in the scholarly discussions on Capeia and to post your ideas and comments on articles published by our authors. Don‚Äôt hesitate to open up a debate, all constructive contributions are welcome!
          </p>
          <p>
            Capeia not only seeks to stimulate discussion but is also committed to spreading the information and ideas that are expressed on this website into the depths of the World Wide Web. So much so that we provide an extra crowdfunding scheme to collect funds with which to provide a reward on a monthly basis for the scientist whose contribution attracts the most attention. Read an article or share it via one of our social media interfaces and you promote its author for our 
            <strong>
              <a
                href="/author-of-the-month"
              >
                "Author of the Month"
              </a>
            </strong>
             award.
          </p>
          <p
            style={
              Object {
                "margin": "4em 0",
                "textAlign": "center",
              }
            }
          >
            <em>
              If you would like to know more about Capeia's mission and why we think that science and society should interact more closely, please continue.
            </em>
          </p>
          <div>
            <h2
              style={
                Object {
                  "textAlign": "center",
                }
              }
            >
              <a
                href="/welcome"
              >
                I
              </a>
               | 
              II
               | 
              <a
                href="/welcome/3"
              >
                III
              </a>
            </h2>
            <div
              className="row"
            >
              <div
                className="col-sm-6"
              >
                <a
                  href="/welcome"
                >
                  ‚Äπ Welcome to Capeia
                </a>
              </div>
              <div
                className="col-sm-6"
                style={
                  Object {
                    "textAlign": "right",
                  }
                }
              >
                <a
                  href="/welcome/3"
                >
                  Our Motivation ‚Ä∫
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sidebarContainer col-lg-3 col-md-4"
    >
      <aside
        className="sidebar"
      />
    </div>
  </div>
</div>
`;
